### PR TITLE
Revisions for Reiwa era

### LINKED ISSUE #11610: Update documentation with name of new Japanese calendar era
# Update documentation with name of new Japanese calendar era

A new era in the Japanese calendars begins on May 1, 2019. Its name, Reiwa, was announced on April 1, 2019. The documentation needs to be updated to reflect the new era and its name.

### PR DESCRIPTION
**Pull request by rpetrusha**
## Revisions for Reiwa era

Describe your changes here.

Fixes #11610 

@tarekgh

### CODE REVIEW

**Review by tarekgh — APPROVED** [review]

**Comment by tarekgh** [comment]
@rpetrusha the changes here looks good to me. I have a couple of questions/points:

- Do we have any place in the docs that list the Japanese era names? would be nice to list the new name there.
- Is it possible we can update the .NET blog too? I believe the information would be very useful there. 

Thanks

File: docs/standard/datetime/working-with-calendars.md [hunk 1]
```diff
@@ -132,13 +132,16 @@ However, there is one important exception. The default (uninitialized) value of
...
+>  The Reiwa era, a new era in the <xref:System.Globalization.JapaneseCalendar> and <xref:System.Globalization.JapaneseLunisolarCalendar>, begins on May 1, 2019. This change affects all applications that use these calendars. See the following articles for more information:
+> - [Handling a new era in the Japanese calendar in .NET](https://devblogs.microsoft.com/dotnet/handling-a-new-era-in-the-japanese-calendar-in-net/), which documents features added to .NET to support calendars with multiple eras and discusses best practies to use when handling multi-era calenars.
```
Comment (mairaw) [inline docs/standard/datetime/working-with-calendars.md]:
```suggestion
> - [Handling a new era in the Japanese calendar in .NET](https://devblogs.microsoft.com/dotnet/handling-a-new-era-in-the-japanese-calendar-in-net/), which documents features added to .NET to support calendars with multiple eras and discusses best practices to use when handling multi-era calendars.
```

File: docs/standard/datetime/working-with-calendars.md [hunk 2]
```diff
@@ -132,13 +132,16 @@ However, there is one important exception. The default (uninitialized) value of
...
+> - [Prepare your application for the Japanese era change](/windows/uwp/design/globalizing/japanese-era-change), which provides information on testing your applications on Windows to ensure their readiness for the era change.
+> - [Summary of new Japanese Era updates for .NET Framework](https://support.microsoft.com/en-us/help/4477957/new-japanese-era-updates-for-net-framework), which lists .NET Framework updates for individual Windows versions that are related to the new Japanese calendar era, notes new .NET Framework features for multi-era support, and includes things to look for in testing your applications.
 
 An era in most calendars denotes an extremely long time period. In the Gregorian calendar, for example, the current era spans more than two millenia. For the <xref:System.Globalization.JapaneseCalendar> and the <xref:System.Globalization.JapaneseLunisolarCalendar>, the two calendars that support multiple eras, this is not the case. An era corresponds to the period of an emperor's reign. Support for multiple eras, particularly when the upper limit of the current era is unknown, poses special challenges.
```
Comment (mairaw) [inline docs/standard/datetime/working-with-calendars.md]:
```suggestion
An era in most calendars denotes an extremely long time period. In the Gregorian calendar, for example, the current era spans more than two millennia. For the <xref:System.Globalization.JapaneseCalendar> and the <xref:System.Globalization.JapaneseLunisolarCalendar>, the two calendars that support multiple eras, this is not the case. An era corresponds to the period of an emperor's reign. Support for multiple eras, particularly when the upper limit of the current era is unknown, poses special challenges. 
```

File: docs/standard/datetime/working-with-calendars.md [hunk 3]
```diff
@@ -199,7 +202,7 @@ In specifying a era to the <xref:System.Globalization.Calendar.ToDateTime(System
```
Comment (mairaw) [inline docs/standard/datetime/working-with-calendars.md]: unrelated to your change: line 197
In specifying a era -> In specifying an era

File: docs/standard/datetime/working-with-calendars.md [hunk 4]
```diff
@@ -199,7 +202,7 @@ In specifying a era to the <xref:System.Globalization.Calendar.ToDateTime(System
...
+Very much like individual calendars have supported date ranges, eras in the <xref:System.Globalization.JapaneseCalendar> and <xref:System.Globalization.JapaneseLunisolarCalendar> classes also have supported ranges. Previously, .NET used strict era range checks to ensure that an era-specific date was within the range of that era. That is, if a date is outside of the range of the specified era, the method throws an <xref:System.ArgumentOutOfRangeException>. Currently, the .NET Framework uses relaxed ranged checking by default. Updates to all versions of the .NET Framework introduced relaxed era range checks; the attempt to instantiate an era-specific date that is outside the range of the specified era "overflows" into the following era, and no exception is thrown.
```
Comment (mairaw) [inline docs/standard/datetime/working-with-calendars.md]: you mention Framework here, but what about Core?

**Review by mairaw — APPROVED** [review]
LGTM but left a few comments for you to consider

**Comment by rpetrusha** [comment]
@tarekgh, I'll update the blog either later today or first thing tomorrow morning. Also, the list of era names is in the documentation for the JapaneseCalendar and JapaneseLunisolarCalendar classes; I've just opened dotnet/dotnet-api-docs#2224 to revise the documentation for the Reiwa era.

**Comment by rpetrusha** [comment]
Closing and reopening to begin new build.

**Pull request merged**